public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchAnnotationQueryRunner.ts

```diff
@@ -34,11 +36,6 @@ export class CloudWatchAnnotationQueryRunner extends CloudWatchRequest {
         type: 'annotationQuery',
         datasource: this.ref,
       })),
-    }).pipe(
-      map((r) => {
-        const frames = toDataQueryResponse(r).data;
-        return { data: frames };
-      })
-    );
```

Comment:
does super.query handle this for us?

Reply:
[yes](https://github.com/grafana/grafana/blob/d410ef343946c8afcd0a1792bdba6fe6f6388e6b/packages/grafana-runtime/src/utils/DataSourceWithBackend.ts#L242-L247)